src/StaticValidator.php

```diff
@@ -39,7 +41,7 @@ public static function setPluginManager(ValidatorPluginManager $plugins = null)
     public static function getPluginManager()
     {
         if (null === static::$plugins) {
-            static::setPluginManager(new ValidatorPluginManager());
+            static::setPluginManager(new ValidatorPluginManager(new ServiceManager));
```

Comment:
@Maks3w note the context here: static usage. By very definition, no instance is available.

Within the MVC and the various plugin manager factories, the SM *will* be injected by default; that's the new design. (Instead of passing it to `setServiceLocator()`, it becomes a constructor argument, and becomes the "creation context", which means that it is that instance, not the plugin manager, passed to factories.)